src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is work");

```

Comment:
The message "Excavator is work" is not informative enough and contains a grammatical mistake. Consider rephrasing it to something like "Excavator started its work" to clearly indicate the action being performed and to use proper English.